README.md

```diff
@@ -46,32 +46,32 @@ Example:
 -- THESE ARE EXAMPLE CONFIGS FEEL FREE TO CHANGE TO WHATEVER YOU WANT
 -- general
 O.auto_complete = true
-O.relative_number = false
 O.colorscheme = 'spacegray'
-O.timeoutlen = 100
 O.leader_key = ' '
+O.relative_number = false
+O.timeoutlen = 100
 
 -- After changing plugin config it is recommended to run :PackerCompile
-O.plugin.hop.active = true
 O.plugin.colorizer.active = true
-O.plugin.trouble.active = true
+O.plugin.hop.active = true
```

Comment:
while you're changing this I would appreciate it if you remove the following as they have been removed from built-in plugins:
```
O.plugin.hop.active = true
O.plugin.lazygit.active = true
O.plugin.markdown_preview.active = true
```

Reply:
fixed